AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -373,6 +407,7 @@ - (void)play
     }
   }
   
+  self.playerState = ASVideoNodePlayerStatePlaying;
```

Comment:
My concern here is, should the player state be set to playing in cases where the buffer is empty? Should we check the item status ready to play on the player item first?